Extension/package.json

```diff
@@ -42,6 +42,168 @@
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. [Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isMac",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing Clang on macOS",
+              "path": "walkthrough/install-clang-macos.md"
+            }
+          },
+          {
+            "id": "verify.compiler.linux",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isLinux",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing GCC on Linux",
+              "path": "walkthrough/install-gcc-linux.md"
+            }
+          },
+          {
+            "id": "verify.compiler.windows",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isWindows",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing MSVC on Windows",
+              "path": "walkthrough/install-compiler-windows.md"
+            }
+          },
+          {
+            "id": "open.project",
+            "title": "Open your project",
+            "description": "Open an existing folder that contains your C++ project, or create a new folder. \n[Open Folder](command:workbench.action.files.openFolder)",
+            "when": "workspaceFolderCount == 0",
+            "doneOn": {
+              "command": "workbench.action.files.openFolder"
+            },
+            "media": {
+              "altText": "screenshot that shows a C++ project opened in the file Explorer",
+              "path": "walkthrough/cpp-folder-explorer.PNG"
+            }
+          },
+          {
+            "id": "setup.intellisense",
+            "title": "Configure C++ IntelliSense",
+            "description": "Verify important IntelliSense settings like Compiler Path, Compiler Args, IntelliSense Mode, and Include Path. \n[Open IntelliSense Configuration](command:C_Cpp.ConfigurationEditUI?-2)",
+            "doneOn": {
+              "command": "C_Cpp.ConfigurationEditUI"
+            },
+            "media": {
+              "altText": "screenshot that shows IntelliSense configurations UI with important settings highlighted",
```

Comment:
Was this Configurations supposed to be capital too to be consistent with the other usage?

Reply:
Yes, updating.